openapi.yaml

```diff
@@ -0,0 +1,1791 @@
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+  /accounts/{accountId}/staking-info:
+    get:
+      tags:
+      - accounts
+      summary: Get staking information for a stash account.
+      description: Returns information about a stash account's staking activity.
+        Replaces `/staking/{address}` from versions < v1.0.0.
+      operationId: getStakingSummaryByAccountId
+      parameters:
+      - name: accountId
+        in: path
+        description: SS58 address of the account.
+        required: true
+        schema:
+          pattern: '^[1-9A-HJ-NP-Za-km-z]{8,64}$'
+          type: string
+      - name: at
+        in: query
+        description:  Block at which to query the staking info for the
+          specified account.
+        required: false
+        schema:
+          type: string
+          description: Block identifier, as the block height or block hash.
+          format: unsignedInteger or $hex
+      responses:
+        "200":
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/AccountStakingInfo'
+        "400":
+          description: invalid blockId supplied for at query param
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        "404":
+          description: Account not found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+  /accounts/{accountId}/vesting-info:
+    get:
+      tags:
+      - accounts
+      summary: Get vesting information for an account.
+      description: Returns the vesting schedule for an account.
+        Replaces `/vesting/{address}` from versions < v1.0.0.
+      operationId: getVestingSummaryByAccountId
+      parameters:
+      - name: accountId
+        in: path
+        description: SS58 address of the account.
+        required: true
+        schema:
+          pattern: '^[1-9A-HJ-NP-Za-km-z]{8,64}$'
+          type: string
+      - name: at
+        in: query
+        description: Block at which to query the vesting info for the
+          specified account.
+        required: false
+        schema:
+          type: string
+          description: Block identifier, as the block height or block hash.
+          format: unsignedInteger or $hex
+      responses:
+        "200":
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/AccountVestingInfo'
+        "400":
+          description: invalid blockId supplied for at query param
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        "404":
+          description: account not found
```

Comment:
There should be a difference between "account doesn't exist" and "account does not have a vesting schedule".

Reply:
@danforbes Do you know how to represent the difference?

Reply:
Ah ok, empty object is fine